parachain/src/primitives.rs

```diff
@@ -186,6 +186,23 @@ impl<T: Encode + Decode + Default> AccountIdConversion<T> for Id {
 	}
 }
 
+/// A type that uniquely identifies an HRMP channel. An HRMP channel is established between two paras.
+/// In text, we use the notation `(A, B)` to specify a channel between A and B. The channels are
+/// unidirectional, meaning that `(A, B)` and `(B, A)` refer to different channels. The convention is
+/// that we use the first item tuple for the sender and the second for the recipient. Only one channel
+/// is allowed between two participants in one direction, i.e. there cannot be 2 different channels
+/// identified by `(A, B)`.
+///
+/// `HrmpChannelId` has a defined ordering: first `sender` and tie is resolved by `recipient`.
+#[derive(Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Hash))]
+pub struct HrmpChannelId {
```

Comment:
If there is a defined ordering worth mentioning in the documentation, why not derive `PartialOrd, Ord`?

Reply:
That's a great catch! Apparently, this property was specified because we needed sorting somewhere but now the impl revealed that the spec didn't actually rely on this.

I'll remove this note